index.js

```diff
@@ -304,6 +305,8 @@ Truth.prototype.clone = function clone(name) {
   var truth = new Truth(name);
 
   truth.transforms = this.transforms.slice();
+  truth.unique = this.unique;
+
```

Comment:
Didn't want to provide `clone` with an options arg here as I thought it looked silly. But yeah that would def be an improvement